cran-comments.md

```diff
@@ -0,0 +1,10 @@
+## Test environments
+* local OS X install, R 4.0.0
```

Comment:
Let's add other test environments, if possible. @CBjerke @DarkShadeKnigh can one of you add windows test environment?
Ideally, someone would test on R 3.1.X